package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "com.padoru.localization-lib",
-  "version": "0.5.1",
+  "version": "0.5.2",
```

Comment:
I believe this is required and, once we merge this we have to update the Unity Package Manager to use this new version instead manually? But let me know if this is not the way we are supposed to change the version of our packages.

Reply:
this is correct